recognition/advanced_tree_search.py

```diff
@@ -145,7 +145,9 @@ def create_config(cls, crp, feature_scorer, extra_config, extra_post_config, **k
     @classmethod
     def hash(cls, kwargs):
         config, post_config, num_images = cls.create_config(**kwargs)
-        return super().hash({"config": config, "exe": kwargs["crp"].flf_tool_exe})
+        return super().hash(
+            {"config": config, "exe": kwargs["crp"].flf_tool_exe, "post_config": post_config},
```

Comment:
You add the post_config to the things that should be hashed?? This changes the hashes of all the setups that use this Job.

Pls. revert this line.

Reply:
Whoops this was a leftover from testing.